src/App.js

```diff
@@ -41,6 +64,8 @@ function App() {
 
   return (
     <>
+    {isLoading?<p>...isLoading</p>:<p></p>}
```

Comment:
this line is exactly what the assignment is asking you to do but I wanted to  show you how this line can be simplified with the concept of [Nullish coalecing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) just FYI.

Reply:
Hello,
Thank you for your suggestions. Fix the ternary operator. But didn't get how we can use [Nullish coalecing]
 because on the left I need Bolian and if I use {isLoading??<TodoList...>}  where isLoading - variable nad if it is true, then I will get "True" and render it but this is not what I need, instead I need "isLoading" string which I can get by normal ternary like first parameter before ":".  Will be glad to know your advise!
Best regards,
Anton